apis/dkjson

```diff
@@ -11,7 +11,7 @@ David Kolf's JSON module for Lua 5.1/5.2
 This module writes no global values, not even the module table.
 Import it using
 
-    json = require ("dkjson")
+    json = require ("github.rom/apis/dkjson")
```

Comment:
I am aware that require is not CC specific, but to say it doesn't exist in CC is incorrect.  However you are right, I'm essentially setting json = true here (since that's what require returns).